doc/level1.rst

```diff
@@ -393,7 +393,13 @@ The Autocrypt recommendation depends on the recipient
 addresses of the draft message.  When the user changes the
 recipients, the Autocrypt recommendation may change.
 
-Autocrypt can produce four possible recommendations:
+The output of the Autocrypt recommendation algorithm has two elements:
+
+ * ``ui-recommendation``: a single state recommending the state of the
+   encryption user interface, described below.
+ * ``target-keys``: a map of recipient addresses to public keys.
+
+``ui-recommendation`` can take four possible values:
 
  * ``disable``: Disable or hide any UI that would allow the user to
```

Comment:
We could mention here that this recommendation happens exactly iff encrypting is in fact not available.